link-check/caasLinks.js

```diff
@@ -0,0 +1,49 @@
+import { compare } from './linkCompare.js';
+import { ExcelReporter, loadListData } from '../bulk-update/index.js';
+
+const { pathname } = new URL('.', import.meta.url);
+const MILO_BLOG = 'https://main--bacom-blog--adobecom.hlx.live';
+const FRANKLIN_BLOG = 'https://main--business-website--adobe.hlx.live';
+const LIST = [
+  'https://main--bacom-blog--adobecom.hlx.live/de/blog/query-index.json',
+  'https://main--bacom-blog--adobecom.hlx.live/fr/blog/query-index.json',
+  'https://main--bacom-blog--adobecom.hlx.live/au/blog/query-index.json',
+  'https://main--bacom-blog--adobecom.hlx.live/uk/blog/query-index.json',
+  'https://main--bacom-blog--adobecom.hlx.live/blog/query-index.json',
+  'https://main--bacom-blog--adobecom.hlx.live/jp/blog/query-index.json',
+  'https://main--bacom-blog--adobecom.hlx.live/kr/blog/query-index.json',
+];
+
+async function init() {
+  const reporter = new ExcelReporter(`${pathname}reports/caas-links.xlsx`, true);
+  const list = await loadListData(LIST);
+  const { length } = list;
+
+  for (const [i, entry] of list.entries()) {
+    const percentage = Math.round(((i + 1) / length) * 10000) / 100;
+    console.log(`Processing entry ${i + 1} of ${length} (${percentage}%) ${entry}`);
+    const miloURL = MILO_BLOG + entry;
+    const franklinURL = FRANKLIN_BLOG + entry;
+    try {
+      const result = await compare(miloURL, franklinURL);
```

Comment:
It looks like compareLink doesn't compare the hashes. Should we include that? A lot of the old pages have the Table of Contents-style bullets with anchor links https://business.adobe.com/fr/blog/basics/8-wastes-of-lean

Reply:
You can now pass a custom function into compare that returns true/false so any logic can be added to the checker.